components/tabstrip/tabs-position.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Tabs Position
+page_title: TabStrip Tabs Position
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs-position
+tags: telerik,blazor,tab,strip,tabstrip,position
+published: True
+position: 10
+---
+
+# Tabs Position
+
+By default, the tab titles are positioned on top of the tabs.
+
+You can customize their position through the optional `TabPosition` attribute the main `TelerikTabStrip` tag exposes. It takes a member of the `Telerik.Blazor.TabPosition` enumeration:
```

Comment:
Use more straight-forward and simple sentence structure. Avoid sequences of subjects and verbs.

```suggestion
You can customize their position through the optional `TabPosition` attribute of the `TelerikTabStrip` tag. It takes a member of the `Telerik.Blazor.TabPosition` enumeration:
```